### PR TITLE
Create Physician Model

### PR DESCRIPTION
**Pull request by michaelwehrley**
Create Physician Model

* Add Physician Class (STI) and migrations.
* Update fixtures to include physicians.
* Add rspec tests for physician.

[Finishes: #92143176]

### CODE REVIEW

File: app/controllers/payloads_controller.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Handles aggregating json data for syncing.
 class PayloadsController < ApplicationController
-  before_action :categories
+  before_action :categories, :chapters, :pages, :physicians
```
Comment (ericcf) [inline app/controllers/payloads_controller.rb]: @michaelwehrley what is the purpose of these callbacks?

File: app/models/physician.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# These physicians are linked to each participant.
+class Physician < ActiveRecord::Base
+  validate :validate_type
+  validates :image_src, :name, presence: true
```
Comment (ericcf) [inline app/models/physician.rb]: @michaelwehrley presence validation will fail for an empty string (http://api.rubyonrails.org/classes/ActiveModel/Validations/HelperMethods.html#method-i-validates_presence_of). However, your default value in the migration is defined as an empty string. Is this discrepancy intentional?

Comment (michaelwehrley) [inline app/models/physician.rb]: Should be all set.

**Comment by michaelwehrley** [comment]
@ericcf Could you take a look at this when you get a chance.  Thanks!

**Comment by ericcf** [comment]
Looks good, merging.

**Pull request merged**